types/ClimateMeasurement.json

```
@@ -0,0 +1,20 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "/types/ClimateyMeasurementType",
+    "description": "Climate Measurements",
+    "type" : "object",
+   
+    "properties" : {
+        "value" : {
+            "type" : "number"
+        },
+        "unit" : {
+            "description" : "UNCEFACT unit",
+            "type" :"string"
+        },
+        "resolution" : {
+            
+            "type" :"number"
```

Possible description (though I'm not sure if it's correct):

```suggestion
            "type" : "number",
            "description": "The number of decimal places of accuracy"
```